src/pages/sentinel/docs/[[...page]].tsx

```diff
@@ -0,0 +1,34 @@
+import { ReactElement } from 'react'
+import sentinelData from 'data/sentinel.json'
+import { Product } from 'types/products'
+import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
+import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
+import DocsView from 'views/docs-view'
+//
+import SentinelEmbedded from '@hashicorp/react-sentinel-embedded'
+import remarkSentinel from 'lib/remark-sentinel'
+import { sentinelUrlAdjuster } from 'layouts/sidebar-sidecar/utils/product-url-adjusters'
```

Comment:
[Non-blocking] Suggestion to remove empty inline comment (`//`) and rearrange the imports a little bit (@hashicorp import right after `react` import, group `layout` imports together, move `lib` import higher in the list).

```suggestion
import { ReactElement } from 'react'
import SentinelEmbedded from '@hashicorp/react-sentinel-embedded'
import sentinelData from 'data/sentinel.json'
import { Product } from 'types/products'
import remarkSentinel from 'lib/remark-sentinel'
import { getStaticGenerationFunctions } from 'layouts/sidebar-sidecar/server'
import SidebarSidecarLayout from 'layouts/sidebar-sidecar'
import { sentinelUrlAdjuster } from 'layouts/sidebar-sidecar/utils/product-url-adjusters'
import DocsView from 'views/docs-view'
```